File: Szw-Todo-MVVM/app/src/main/java/ca/six/todo/SplashActivity.java
package ca.six.todo;

import android.app.Activity;
import android.os.Bundle;
import android.os.Handler;
import android.os.Message;

// 本页不宜用android:noHistory. 因为要防止press home key
public class SplashActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);


    }


}

File: deprecated/Tools/DaggerRxJavaProject/app/src/main/java/ca/six/demo/biz/splash/SplashActivity.kt
package ca.six.demo.biz.splash

import android.content.Intent
import android.os.Bundle
import android.view.MotionEvent
import ca.six.demo.R
import ca.six.demo.biz.home.HomeActivity
import ca.six.demo.core.BaseActivity


class SplashActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        println("szw SplashPage : $retrofit")

    }

    override fun onTouchEvent(event: MotionEvent): Boolean {
        if (MotionEvent.ACTION_UP == event.action) {
            startActivity(Intent(this, HomeActivity::class.java))
        }
        return super.onTouchEvent(event)
    }

}

